ansible/roles/de-TMP-RUN-ONCE-app-beta-tagger/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- debug:
+    msg: "This role should only be RUN ONCE against the metadata database. It will fail if run more than once."
```

Comment:
The tool could probably be improved to check for existing beta AVUs, and then not attempt to insert one if found. This would be similar to the logic in the metadata POST /avus endpoint.